docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -292,6 +292,47 @@ The individual match elements are: -
 * <match-domain-name name="..."/>:: Matches where the protocol of the URI is 'domain' and the scheme specific part of the URI is the name specified within this match element.
  <match-abstract-type name="..." authority="..." />:: Matches the abstract type and/or authority against the values specified within this match element.
 
+=== Elytron client default SSL context security provider
+
+To make use of the `org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider`, a runtime dependency on `wildfly-elytron-client` and `wildfly-client-config` is needed. This java security provider loads Elytron client configuration and provides an initialized default SSL context. You can make use of this SSL context by calling `SSLContext.getDefault()` if this provider is registered with high enough priority.
```

Comment:
> To make use of the `org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider`, a runtime dependency on `wildfly-elytron-client` and `wildfly-client-config` is needed

I've noticed how this was introduced and it seems to me that context is missing from the user PoV.
First of all, this is an implementation detail, which IMHO should come after the preface, i.e. what is being introduced is the default SSL context security provider, then we mention that if users want to use it in their application, they need to add such dependencies directly to their project.

This is not blocking the PR from being merged but IMO should be fixed.

Reply:
Thanks @Skyllarr - LGTM, but I noticed @fjuma suggested some rewording, which seems even clearer to me.